SimulationRuntime/c/simulation/solver/solver_main.c

```diff
@@ -465,6 +465,8 @@ int initializeModel(DATA* data, threadData_t *threadData, const char* init_initM
       simInfo->stopTime = simInfo->startTime;
       retValue = -1;
     }
+    if (!retValue)
+      infoStreamPrint(LOG_STDOUT, 0, "The initialization finished successfully %s homotopy method.", data->simulationInfo->homotopyUsed ? "with" : "without");
```

Comment:
Don't log this to STDOUT; if you do we can't disable it in the logging settings

Reply:
We can add an option to disable `LOG_STDOUT` or we can have a new default notification stream that is enabled by default – which is then basically the same as having an option to disable `LOG_STDOUT`.

Reply:
I also thought about these two possibilities. We could have the option `-lv=NO_STDOUT`.

Reply:
Don't touch stdout. This belongs to LOG_INIT or a separate category.